src/FileHandler/FileHandler.java

```diff
@@ -0,0 +1,461 @@
+package FileHandler;
+
+import model.Video;
+import service.VideoService;
+import strategy.SearchStrategy;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileHandler {
+
+    public void adicionarVideo (Scanner scanner, VideoService videoService) {
+        String titulo = "";
+        String descricao = "";
+        int duracao = 0;
+        String categoria = "";
+        String dataStr;
+        int escolhaCategoria;
+        do {
+
+            try {
+                System.out.print("Digite o título do vídeo: ");
+                titulo = scanner.nextLine();
+                System.out.println(" ");
+
+                if (titulo.isBlank()) {
+                    System.out.println(" --- Por favor, escreva um título! --- ");
+                    System.out.println(" ");
+                    continue;
+                }
+                if (titulo.charAt(0) >= '0' && titulo.charAt(0) <= '9') {
```

Comment:
Existe uma função que verifica se o char é um digito.
```suggestion
                if (Character.isDigit(titulo.charAt(0))) {
```

Reply:
Implementado!